test/mcp/tools_test.exs

```diff
@@ -10,6 +10,23 @@ defmodule Tidewave.MCP.ToolsTest do
     end
   end
 
+  test "tools are hidden from the list when excluded" do
+    refute Enum.any?(
+             Tidewave.MCP.Server.tools({%{}, [tools: [exclude: ["project_eval"]]]}),
```

Comment:
do we expect passing tools as atoms, since you do `&to_string/1` in init? If we do, we should have one assertion here that doesn't already use strings

Reply:
That test would need to be in the tests for the plug. I actually had some trouble figuring out how best to test it at that layer, because I'd need to test the response of the list tools and there wasn't an example of that, and when I tried it I didn't get the response I expected. I would call the list tools action and just get the 200 ok response w/ no tools. Any ideas?